src/main/scala/org/broadinstitute/dsde/workbench/avram/dataaccess/HttpSamDao.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.workbench.avram.dataaccess
+
+import java.util.logging.Logger
+
+import com.softwaremill.sttp._
+import io.circe.Decoder.Result
+import io.circe.{DecodingFailure, Json, ParsingFailure}
+import io.circe.generic.auto._
```

Comment:
Oh, BTW, this is really cool! Note the lack of encode/decode for , or annotations on, `UserInfo`.

Reply:
That is very cool